From e0b1abaf882e5d05c15bab8a1b5225474b0f9492 Mon Sep 17 00:00:00 2001
From: Umberto Baldi <u.baldi@arduino.cc>
Date: Tue, 31 May 2022 16:14:48 +0200
Subject: [PATCH] Added fake EFUSE to atmega8 part. This allows to have a
 common platform.txt recipe for all the ATMegaXX parts in the Arduino IDE.

See:
https://github.com/arduino/Arduino/issues/2075#issuecomment-238031689
https://github.com/arduino/Arduino/issues/2075


Co-authored-by: Martino Facchin <m.facchin@arduino.cc>
---
 src/avrdude.conf.in | 7 +++++++
 1 file changed, 7 insertions(+)

diff --git a/src/avrdude.conf.in b/src/avrdude.conf.in
index 362b616..792d603 100644
--- a/src/avrdude.conf.in
+++ b/src/avrdude.conf.in
@@ -6832,6 +6832,13 @@ part
                           "x x x x  x x x x   i i i i  i i i i";
       ;
 
+    # Required for Arduino IDE
+    # see: https://github.com/arduino/Arduino/issues/2075
+    #      https://github.com/arduino/Arduino/issues/2075#issuecomment-238031689
+    memory "efuse"
+        size            = 0;
+      ;
+
     memory "lock"
         size            = 1;
         min_write_delay = 2000;
-- 
2.17.1

